src/components/NetworkSettings/NetworkStatusBar.js

```diff
@@ -0,0 +1,22 @@
+import { useSelector } from "react-redux";
+import { AlertUI } from "@styles/themes";
+import { ToplineBar } from "@components/ToplineBar";
+import { eq } from "lodash";
+import { PRE_SETTINGS_MAINNET, PRE_SETTINGS_TESTNET } from "../../constants";
+
+export const NetworkStatusBar = () => {
+  const networkSettings = useSelector((state) => state.networkSettings);
+  if (eq(networkSettings , PRE_SETTINGS_MAINNET)
+    || eq(networkSettings, PRE_SETTINGS_TESTNET)) {
```

Comment:
I think we should show the status bar for testnet as well.

Reply:
- Show to testnet: 4c36c756214e8faad39d9106035acf86eeb84775